### ISSUE TITLE
automatic_rename_kit_nodes causes: list assignment index out of range

### ISSUE THREAD

**Issue by Cynerd** [bigclownlabs/bch-gateway#9]
Latest version of gateway fails to start with following error:
```
root@turris-prague:~# bcg -c /tmp/etc/bigclown-gateway.conf 
2019-02-27 10:35:08,756 ERROR: list assignment index out of range
```

I am not sure it this is a bug or if this is no longer supported option, but either way it is still provided by configuration examples.

This is generated config:
```
root@turris-prague:~# cat /tmp/etc/bigclown-gateway.conf 
name: usb-dongle
device: /dev/ttyUSB0
automatic_rename_kit_nodes: 1
mqtt:
  host: localhost
  port: 1883
```

**Comment by blavka** [comment]
Thx for report.

**Comment by blavka** [comment]
New bcg version with fix is out v1.16.2

**Comment by Cynerd** [comment]
Thx
